src/sentry/integrations/slack/unfurl/__init__.py

```diff
@@ -23,7 +24,10 @@ class UnfurlableUrl(NamedTuple):
 class Handler(NamedTuple):
     matcher: Pattern[Any]
     arg_mapper: ArgsMapper
-    fn: Callable[[HttpRequest, Integration, List[UnfurlableUrl]], UnfurledUrl]
+    fn: Union[
```

Comment:
Instead of putting a `Union` in the `Callable` can you add an unused parameter to the other handlers like:
```python
def unfurl_issues(
    request: HttpRequest,
    integration: Integration,
    links: List[UnfurlableUrl],
    identity: Optional["User"] = None
) -> UnfurledUrl:
  ...
```